pkg/router/template/template_helper.go

```diff
@@ -97,7 +97,7 @@ func genSubdomainWildcardRegexp(hostname, path string, exactPath bool) string {
 }
 
 // Generate a regular expression to match route hosts (and paths if any).
-func generateRouteRegexp(hostname, path string, wildcard bool) string {
+func generateRouteRegexp(hostname, path string, pathRegexp bool, wildcard bool) string {
```

Comment:
At the very least, the existing function cannot be modified for backward compatibility reasons.

Reply:
I’ll add a second function, then.

Reply:
Handled. The new function is called `generateRouteRegexp2`. `generateRouteRegexp` calls out to the new function with `pathRegexp` set to `false`, preserving the current behavior.